network-store-server/src/test/java/com/powsybl/network/store/server/NetworkStoreControllerIT.java

```diff
@@ -886,10 +869,6 @@ void test() throws Exception {
                         .contentType(APPLICATION_JSON)
                         .content(objectMapper.writeValueAsString(Collections.singleton(tieLine))))
                 .andExpect(status().isOk());
-
-        mvc.perform(delete("/" + VERSION + "/networks/" + NETWORK_UUID + "/" + Resource.INITIAL_VARIANT_NUM + "/tie-lines/idTieLine")
-                .contentType(APPLICATION_JSON))
-                .andExpect(status().isOk());
```

Comment:
Can we also keep the delete for this one?

Reply:
done